doc/source/index.rst

```diff
@@ -8,16 +8,16 @@ SciPy documentation
 
 **Date**: |today| **Version**: |version|
 
-**Download documentation**: `PDF Version <https://docs.scipy.org/doc/scipy-1.6.1/scipy-ref-1.6.1.pdf>`__ | `Zipped HTML <https://docs.scipy.org/doc/scipy-1.6.1/scipy-html-1.6.1.zip>`__
+**Download documentation**: `PDF Version <https://docs.scipy.org/doc/scipy-1.6.2/scipy-ref-1.6.2.pdf>`__ | `Zipped HTML <https://docs.scipy.org/doc/scipy-1.6.2/scipy-html-1.6.2.zip>`__
```

Comment:
This is going to be outdated again soon. I suggest replacing the direct links with a single link to https://docs.scipy.org/doc/

Reply:
Thanks, I was not sure what to put here. I wanted to use `|version|` but could not make it work.

Reply:
Thanks, I was not sure what to put here. I wanted to use `|version|` but could not make it work.